com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -94,29 +82,6 @@ public void UpdateLinkData()
             }
         }
 
-        private void OnDrawGizmosSelected()
-        {
-            if (displayInertiaGizmo)
-            {
-                #if UNITY_2020_1_OR_NEWER
-                    Debug.Log("'ArticulationBody' does not contain a definition for 'inertiaTensorRotation' and no accessible extension method 'inertiaTensorRotation'");
```

Comment:
Is it just me, or does the comment not make sense? ABs have a public property called inertiaTensor and inertiaTensorRotation?

Reply:
We didn't have it back in 2020.1 (in the initial AB roll out).